### PR TITLE
Document Apache Guacamole release procedures.

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: release-procedure-part2.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+hosted under the Apache Guacamole project's [Docker Hub
+account](http://guacamole.incubator.apache.org/releases/).
+
+When building the Docker images, keep in mind:
+
+1. You should run `git clean -xfd .` first, to ensure the build context
+   uploaded to the Docker daemon is minimal, and that future builds of the
+   exact same source will not result in different images (this serves as a
+   pre-release sanity check).
+2. You **MUST NOT** update the `latest` tag on Docker Hub, as this must only
```
Comment (jmuehlner) [inline release-procedure-part2.md]: Wouldn't the new release BE the latest stable release? I'm not sure I get this.

Comment (mike-jumper) [inline release-procedure-part2.md]: When it's out, yes, but at this point in the process the new release is still a release candidate.

Comment (jmuehlner) [inline release-procedure-part2.md]: Ah yes, there it is further down. 👍

**Comment by jmuehlner** [comment]
Super comprehensive docs! :shipit:

**Pull request merged**